scripts/release/list_issues.py

```diff
@@ -147,10 +147,10 @@ def parse_args():
 def main():
     parse_args()
 
-    token = os.environ.get('GH_TOKEN', None)
+    token = os.environ.get('GITHUB_TOKEN', None)
```

Comment:
These changes are not mentioned in the commit log and also seem like they should be in their own commit.

Reply:
I was asked to include them here by Stephanos. Would be fine to mention it in the commit msg.

Reply:
I see that the change was requested, but I see no request (or reason for one) that it has to be in this same commit; is there one?